d_grpc_bench/Dockerfile

```diff
@@ -1,22 +1,33 @@
-FROM dlang2/ldc-ubuntu:1.26.0 AS builder
+# syntax=docker/dockerfile:1
 
-RUN apt update && apt install -y protobuf-compiler git cmake g++
+FROM buildpack-deps:bookworm AS builder
+
+RUN set -eux; \
+    wget -O ldc2-1.40.1-linux-x86_64.tar.xz https://github.com/ldc-developers/ldc/releases/download/v1.40.1/ldc2-1.40.1-linux-x86_64.tar.xz; \
+    echo "085a593dba4b1385ec03e7521aa97356e5a7d9f6194303eccb3c1e35935c69d8 *ldc2-1.40.1-linux-x86_64.tar.xz" | sha256sum -c -; \
+    tar --strip-components=1 -C /usr/local -Jxf ldc2-1.40.1-linux-x86_64.tar.xz; \
+    rm ldc2-1.40.1-linux-x86_64.tar.xz
+
+RUN set -eux; \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+        protobuf-compiler \
+        git \
+        cmake \
+        g++; \
+    rm -fr /var/lib/apt/lists/*
 
-WORKDIR /app
-RUN git clone --depth 1 --branch master --recurse-submodules https://github.com/huntlabs/grpc-dlang
-WORKDIR /app/grpc-dlang
 # Building the protocol buffer compiler for D
 RUN dub build protobuf:protoc-gen-d
+
+WORKDIR /app
+RUN git clone --depth 1 --branch master --recurse-submodules https://github.com/huntlabs/grpc-dlang
```

Comment:
I know it was already there, but would you mind locking a specific  commit so  that it doesn't break all of a sudden?